packages/keyring-api/src/btc/types.ts

```diff
@@ -41,6 +49,11 @@ export const BtcP2wpkhAccountStruct = object({
    */
   type: literal(`${BtcAccountType.P2wpkh}`),
 
+  /**
+   * Account supported scopes (CAIP-2 chain IDs).
+   */
+  scopes: nonempty(array(CaipChainIdStruct)), // TODO: Would be great to further restrict this to be an array of 1 element?
```

Comment:
I think it makes sense to add the restriction mentioned in the TODO. Any reason to not do it?

Reply:
No actual reason no, I'll check if `superstruct` has something for this, otherwise I might to write this logic myself!

Reply:
Actually, `size` should work, I just missed it when looking at `superstruct` lemme change that now!